package.json

```diff
@@ -219,6 +220,7 @@
 		"@wordpress/eslint-plugin": "^12.3.0",
 		"babel-loader": "^8.2.3",
 		"bunyan": "^1.8.15",
+		"canvas": "^2.9.3",
```

Comment:
> **If you have an ARM CPU**
>
> Prebuilds aren't available for ARM. Please see the [wiki](https://github.com/Automattic/node-canvas/wiki#installation-guides) for guides on building from source. See also https://github.com/node-gfx/node-canvas-prebuilt/issues/69.

This library does not appear to publish a prebuild for M1 Macs with ARM chips. This caused me issue when attempting to install this package. I ultimately installed additional dependencies so that building from source succeeded when running `yarn install`. Did you encounter this? 

<details><summary>Error Text</summary>

```
# This file contains the result of Yarn building a package (canvas@npm:2.9.3)
# Script name: install

node-pre-gyp info it worked if it ends with ok
node-pre-gyp info using node-pre-gyp@1.0.9
node-pre-gyp info using node@16.13.2 | darwin | arm64
node-pre-gyp http GET https://github.com/Automattic/node-canvas/releases/download/v2.9.3/canvas-v2.9.3-node-v93-darwin-unknown-arm64.tar.gz
node-pre-gyp ERR! install response status 404 Not Found on https://github.com/Automattic/node-canvas/releases/download/v2.9.3/canvas-v2.9.3-node-v93-darwin-unknown-arm64.tar.gz
node-pre-gyp WARN Pre-built binaries not installable for canvas@2.9.3 and node@16.13.2 (node-v93 ABI, unknown) (falling back to source compile with node-gyp)
node-pre-gyp WARN Hit error response status 404 Not Found on https://github.com/Automattic/node-canvas/releases/download/v2.9.3/canvas-v2.9.3-node-v93-darwin-unknown-arm64.tar.gz
gyp info it worked if it ends with ok
gyp info using node-gyp@8.2.0
gyp info using node@16.13.2 | darwin | arm64
gyp info ok
gyp info it worked if it ends with ok
gyp info using node-gyp@8.2.0
gyp info using node@16.13.2 | darwin | arm64
gyp info find Python using Python version 3.10.6 found at "/opt/homebrew/opt/python@3.10/bin/python3.10"
gyp info spawn /opt/homebrew/opt/python@3.10/bin/python3.10
gyp info spawn args [
gyp info spawn args   '/Users/username/Sites/a8c/wp-calypso/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/username/Sites/a8c/wp-calypso/node_modules/canvas/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/username/Sites/a8c/wp-calypso/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/username/Library/Caches/node-gyp/16.13.2/include/node/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=/Users/username/Library/Caches/node-gyp/16.13.2',
gyp info spawn args   '-Dnode_gyp_dir=/Users/username/Sites/a8c/wp-calypso/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=/Users/username/Library/Caches/node-gyp/16.13.2/<(target_arch)/node.lib',
gyp info spawn args   '-Dmodule_root_dir=/Users/username/Sites/a8c/wp-calypso/node_modules/canvas',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.'
gyp info spawn args ]
Package cairo was not found in the pkg-config search path.
Perhaps you should add the directory containing `cairo.pc'
to the PKG_CONFIG_PATH environment variable
No package 'cairo' found
gyp: Call to 'pkg-config cairo --libs' returned exit status 1 while in binding.gyp. while trying to load binding.gyp
gyp ERR! configure error
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/Users/username/Sites/a8c/wp-calypso/node_modules/node-gyp/lib/configure.js:353:16)
gyp ERR! stack     at ChildProcess.emit (node:events:390:28)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)
gyp ERR! System Darwin 21.6.0
gyp ERR! command "/Users/username/.asdf/installs/nodejs/16.13.2/bin/node" "/Users/username/Sites/a8c/wp-calypso/node_modules/node-gyp/bin/node-gyp.js" "configure" "--fallback-to-build" "--update-binary" "--module=/Users/username/Sites/a8c/wp-calypso/node_modules/canvas/build/Release/canvas.node" "--module_name=canvas" "--module_path=/Users/username/Sites/a8c/wp-calypso/node_modules/canvas/build/Release" "--napi_version=8" "--node_abi_napi=napi" "--napi_build_version=0" "--node_napi_label=node-v93"
gyp ERR! cwd /Users/username/Sites/a8c/wp-calypso/node_modules/canvas
gyp ERR! node -v v16.13.2
gyp ERR! node-gyp -v v8.2.0
gyp ERR! not ok
node-pre-gyp ERR! build error
node-pre-gyp ERR! stack Error: Failed to execute '/Users/username/.asdf/installs/nodejs/16.13.2/bin/node /Users/username/Sites/a8c/wp-calypso/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --update-binary --module=/Users/username/Sites/a8c/wp-calypso/node_modules/canvas/build/Release/canvas.node --module_name=canvas --module_path=/Users/username/Sites/a8c/wp-calypso/node_modules/canvas/build/Release --napi_version=8 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v93' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/Users/username/Sites/a8c/wp-calypso/node_modules/@mapbox/node-pre-gyp/lib/util/compile.js:89:23)
node-pre-gyp ERR! stack     at ChildProcess.emit (node:events:390:28)
node-pre-gyp ERR! stack     at maybeClose (node:internal/child_process:1064:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:301:5)
node-pre-gyp ERR! System Darwin 21.6.0
node-pre-gyp ERR! command "/Users/username/.asdf/installs/nodejs/16.13.2/bin/node" "/Users/username/Sites/a8c/wp-calypso/node_modules/@mapbox/node-pre-gyp/bin/node-pre-gyp" "install" "--fallback-to-build" "--update-binary"
node-pre-gyp ERR! cwd /Users/username/Sites/a8c/wp-calypso/node_modules/canvas
node-pre-gyp ERR! node -v v16.13.2
node-pre-gyp ERR! node-pre-gyp -v v1.0.9
node-pre-gyp ERR! not ok
Failed to execute '/Users/username/.asdf/installs/nodejs/16.13.2/bin/node /Users/username/Sites/a8c/wp-calypso/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --update-binary --module=/Users/username/Sites/a8c/wp-calypso/node_modules/canvas/build/Release/canvas.node --module_name=canvas --module_path=/Users/username/Sites/a8c/wp-calypso/node_modules/canvas/build/Release --napi_version=8 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v93' (1)
```

</details>

**Related Issues:**

* https://github.com/Automattic/node-canvas/issues/1511
* https://github.com/Automattic/node-canvas/issues/2078
* https://github.com/Automattic/node-canvas/issues/1662

Since we are only adding this for the Jest testing environment, I already wondered if [`jest-canvas-mock`](https://www.npmjs.com/package/jest-canvas-mock) might be a more appropriate package addition. However, these install errors may be another reason to switch. WDYT?

Reply:
Oh, thanks for letting me know. 🤔 I didn't encounter the error you're seeing but the fact that there's so many mixed reports in those linked threads leads me to agree with replacing. You're right that `jest-canvas-mock` is likely more appropriate, too. I've gone ahead to replace the dependency in https://github.com/Automattic/wp-calypso/pull/67013/commits/d9ac75e4f9acdfe79d2cd75ce4d05ee7e68c6f1c.